firewood/src/merkle.rs

```diff
@@ -2095,7 +2099,6 @@ mod tests {
     }
 
     // #[test]
```

Comment:
These tests were left over since the rewrite and have never been re-enabled. They need evaluation to see if they add value (that is, they test something that we might not catch with one of the other tests) and enabled if they do, or deleted if they don't.

I doubt there is a ticket for this. If not, let's create one.